assets/schema_gvcfs.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/fellen31/skierfe/master/assets/schema_gvcfs.json",
-    "title": "fellen31/skierfe pipeline - params.extra_gvcfs schema",
+    "$id": "https://raw.githubusercontent.com/fellen31/nallo/master/assets/schema_gvcfs.json",
```

Comment:
```suggestion
    "$id": "https://raw.githubusercontent.com/genomic-medicine-sweden/nallo/master/assets/schema_gvcfs.json",
```

Reply:
Can't find this file on the master branch of this repo but it is there on dev

Reply:
Hm, there's no master branch on fellen31/skierfe anymore since it's now a fork of this repo, but the file still exists there `https://raw.githubusercontent.com/fellen31/skierfe/master/assets/schema_gvcfs.json` but not on the gms master. 

For this PR to pass the tests after name change it would have to point to the dev branch. Or the file first needs to be pushed to the master branch? Will soon have the same problem for all test data.

Reply:
Can take this in the release PR along with the other test-data.